contracts/deploy/999_fork_test_setup.js

```diff
@@ -7,6 +7,7 @@ const {
 const { fundAccounts } = require("../utils/funding");
 const addresses = require("../utils/addresses");
 const daiAbi = require("../test/abi/dai.json").abi;
+const { hardhatSetBalance } = require("../test/_fund");
```

Comment:
I also moved `hardhatSetBalance` into a util file on one of my branches. I can manage to consolidate it when I merge this back into my branches

Reply:
thanks! whoever merges first wins :)